tensorboard/plugins/histogram/summary_v2.py

```diff
@@ -271,14 +271,17 @@ def histogram_pb(tag, data, buckets=None, description=None):
     bucket_count = DEFAULT_BUCKET_COUNT if buckets is None else buckets
     data = np.array(data).flatten().astype(float)
     if data.size == 0:
-        buckets = np.array([]).reshape((0, 3))
+        histogram_buckets = np.array([]).reshape((0, 3))
```

Comment:
For this case, we should match `_buckets_v3` in by returning the all-zeros array of shape `[bucket_size, 3]`: https://github.com/tensorflow/tensorboard/blob/062a59aa6031549fceaed37ffef2ec9806bec468/tensorboard/plugins/histogram/summary_v2.py#L440-L448

That ensures that the size of the resulting array is always `[bucket_size, 3]` no matter what the size or shape of `data` is.

Can you update the docstring accordingly?  Specifically where it says "The output will have this many buckets, except in two edge cases." - that sentence is what we're trying to get rid of here :)  Now, the output shape will always be `[k, 3]` unconditionally.  Rather than affecting the shape, the edge cases affect the contents - i.e. empty data means the edges and counts are all 0, and only a single value means the edges are all that value and only the last bucket has nonzero count (as you've described).

Reply:
Done. Thanks!